Votersimulator/voter.rb

```diff
@@ -0,0 +1,82 @@
+
+require'/Users/MARKIII/Wynprojects/Votersimulator/person_class.rb'
+class Voter
+  attr_accessor :first_name, :second_name, :full_name, :affiliation, :voter_database
+# def initialize
```

Comment:
need to add initializer with @full_name and @affilition, so that all the instances are automatically saved